apis/apiextensions/fn/proto/v1beta1/run_function.pb.go

```diff
@@ -15,7 +15,7 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.31.0
```

Comment:
check-diff fails because of this 🤔 you probably have 1.33 locally and for some reason it's using that instead of the one specified in the go.mod, just revert this change and check-diff will pass